master/docs/manual/cfg-reporters.rst

```diff
@@ -426,7 +426,8 @@ Patches are very welcome for restoring the full functionality.
 
 .. note:: Security Note
 
-Please note that any user having access to your irc channel or can PM the bot will be able to create or stop builds :bug:`3377`.
+    Please note that any user having access to your irc channel or can PM the
```

Comment:
We have a one sentence per line policy, which helps for the review and the diffs please dont break sentences